api/python/setup.py

```diff
@@ -40,6 +40,11 @@
     'pytest-xdist==1.29.0',
     'pytest-timeout==1.3.4',
     'pytest-mock==3.1.0',
+    'attrs==19.1.0',
+    'PyYAML==5.1.2',
```

Comment:
please remove, PyYAML is one of our main dependencies already

Reply:
Done.